src/styles/container.scss

```diff
@@ -0,0 +1,9 @@
+.container {
```

Comment:
Wrong styling:

Check the task guidelines, it has specific asking for this styling:
- Different grid column count based on screen size,
- Gap between columns and rows,
- Fixed padding is 50px 40px,

None of these stylings match here, please review the guidelines and implement them accordingly.
*Hint: for conditional styling depending on the screen size, you can use media queries